src/style.css

```
@@ -0,0 +1,36 @@
+html {
+  background-color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+#qa-block {
+  margin-top: 20px;
+}
+
+.border {
+  border-width: 1px;
+  border-color: #e95d49;
+  border-style: solid;
+}
```

`border` is not the best name for class semantically.
Use 'block' or something similar.

Also, you can use shorthand border property here:

```suggestion
.block {
  border: 1px solid #e95d49;
}
```